recipes/cufftmp/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set version = "11.4.0.6" %}
+{% set libnvshmem_soname = "3" %}
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set extension = "tar.xz" %}  # [unix]
+
+{% set cuda_compiler_version = cuda_compiler_version|default("None") %}
+
+{% set soname = version.split(".")[0] %}
+{% set sha256 = "516714b997bba8266e98ea8cebb3267c33d382d2cccb42bd1731eff3f2846a67" %}  # [linux64]
+{% set sha256 = "" %}  # [aarch64]
+
+package:
+  name: libcufftmp-split
+  version: {{ version }}
+
+source:
+  url: libcufftmp-linux-x86_64-11.4.0.6_cuda12-archive.tar.xz  # [linux64]
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [(cuda_compiler_version in (None, "None", "11.8")) or (not (linux64 or aarch64))]
+  script:
+    - cp -rvp include $PREFIX/
+    - cp -rvp lib $PREFIX/
+
+requirements:
+  build:
+    - cf-nvidia-tools 1  # [linux]
+
+outputs:
+
+  - name: libcufftmp{{ soname }}
+    build:
+      ignore_run_exports_from:
+        - libnvshmem-dev
+    files:
+      - lib/libcufftMp.so.*
+      - bin/chech-glibc lib/libcufftMp.so.*
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+      host:
+        - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
```

Comment:
This recipe needs to depend on `{{ compiler('cuda') }}` if to contains device code. If it is only host code, then it doesn't need cuda-version.